fs-extra/fs-extra-tests.ts

```diff
@@ -101,3 +102,47 @@ fs.ensureSymlink(path, errorCallback);
 fs.ensureSymlinkSync(path);
 fs.emptyDir(path, errorCallback);
 fs.emptyDirSync(path);
+
+const items: Array<string> = [ ]
+fs.walk(path)
+  .on('data', item => {
+    items.push(item.path)
+  })
+  .on('end', function () {
+
+  })
+
+const ignoreHiddenFiles = (item: string): boolean => {
+  const basename = Path.basename(item)
+  return basename === '.' || basename[0] !== '.'
+}
+
+const sortPaths = (left: string, right: string): number => {
```

Comment:
Tests don't actually run, so only the types matter. So this could just be (left: string, right: string) => left.localeCompare(right)`.